employees/views.py

```diff
@@ -220,3 +222,21 @@ def form_valid(self, form: AdminReportForm) -> HttpResponseRedirectBase:
         self.object.editable = True
         self.object.save()
         return super().form_valid(form)
+
+
+@method_decorator(login_required, name="dispatch")
+class ProjectReportList(ListView):
+    template_name = "employees/project_report_list.html"
+    model = Report
+
+    def get_queryset(self) -> QuerySet:
+        return Report.objects.filter(project__id=self.kwargs["pk"]).order_by("-date", "project__name")
+
+    def get_context_data(self, **kwargs: Any) -> dict:
+        context = super().get_context_data(**kwargs)
+        context["UI_text"] = ProjectReportListStrings
+        context["project_name"] = get_object_or_404(Project, pk=self.kwargs["pk"])
```

Comment:
When https://github.com/Code-Poets/sheetstorm/pull/133/files#r283077801 is applied you don't need this, because you have project as `object`.

Reply:
Done